www/contrib.txt

```diff
@@ -0,0 +1,73 @@
+Users contributions
+===================
+
+How to contribute?
+------------------
+
+* Clone the repo with +git clone https://github.com/herbstlufwm/herbstluftwm+
+* Open the +herbstluftwm/www/contrib.txt+ file.
+* Edit the file and add your contribution! It uses the https://powerman.name/doc/asciidoc[asciidoc format].
+* Send your pull request! If you want to test the site run the +make+ command (it needs asciidoc installed). Just remember to run +make clean+ before sending the pull request.
```

Comment:
Actually, I don't see why `make clean` is necessary before sending the pull request :-)

Reply:
In the repo, there's only text files and no html file. In order to mantain it like that I added that part, but is not necessary